pkg/auth/providers/publicapi/login.go

```diff
@@ -208,9 +213,61 @@ func (h *loginHandler) createLoginToken(request *types.APIContext) (v3.Token, st
 		if err != nil {
 			return v3.Token{}, "", "", err
 		}
+
+		if err := h.createClusterAuthTokenIfNeeded(token, tokenValue); err != nil {
```

Comment:
remove `IfNeeded` from the name

Reply:
Doesn't that help explain that its not always created ? If this was a cluster with ACE off it wouldn't be created.

Reply:
Maybe this is a pattern I am not familiar with, I thought IfNeeded didn't give additional context about ACE. So I was thinking createClusterAuthToken with comments or createClusterAuthTokenForAce - okay with keeping it as is too!

Reply:
I feel like createClusterAuth means it will definitely be created, which isn't true here.